app/services/hyrax/multiple_membership_checker.rb

```diff
@@ -55,9 +55,12 @@ def single_membership_collections(collection_ids)
 
       field_pairs = {
         :id => collection_ids,
-        Hyrax.config.collection_type_index_field.to_sym => collection_type_gids_that_disallow_multiple_membership
+        Hyrax.config.collection_type_index_field.to_sym => collection_type_gids_that_disallow_multiple_membership&.map(&:to_s)
```

Comment:
The map to strings is required to turn GlobalIDs into string IDs.  I'm not sure why this worked before.  It definitely does not work with the SolrQueryService.